### PR TITLE
add SMSUnsubscription webhook

### PR DESCRIPTION
**Pull request by PetiHorvath**

### CODE REVIEW

File: openapi/2023-12.yaml [hunk 1]
```diff
@@ -473,4 +473,24 @@ components:
...
+              example: "2021-08-10T14:00:00.000+02:00"
+    SMSUnsubscription:
+      type: object
+      required:
+        - type
+        - attributes
+      properties:
+        type:
+          type: string
+          example: "subscription"
```
Comment (matehuszarik) [inline openapi/2023-12.yaml:485]:
```suggestion
          example: "subscriptions"
```

File: openapi/2023-12.yaml [hunk 2]
```diff
@@ -365,15 +365,15 @@ components:
           properties:
             topic:
               type: string
-              description: Event type, messages/incoming, messages/outgoing
+              description: Event type, messages/incoming, messages/outgoing, sms/unsubscription
```
Comment (matehuszarik) [inline openapi/2023-12.yaml:368]:
```suggestion
              description: Event type, messages/incoming, messages/outgoing, subscribers/unsubscription
```

File: openapi/2023-12.yaml [hunk 3]
```diff
@@ -82,7 +82,7 @@ paths:
           $ref: "#/components/responses/HttpError"
         "5XX":
           $ref: "#/components/responses/HttpError"
-      callbacks: 
+      callbacks:
```
Comment (matehuszarik) [inline openapi/2023-12.yaml:85]: You have to add it here as a callback too

File: openapi/2023-12.yaml (hunk 1 above)
Comment (PetiHorvath) [reply openapi/2023-12.yaml:485]: I think it should be single. We will send back data for 1 subscription.

**Review by matehuszarik — APPROVED** [review]

**Pull request merged**